pgrx/src/spi/query.rs

```diff
@@ -262,9 +270,7 @@ impl<'conn: 'stmt, 'stmt> Query<'conn> for &'stmt PreparedStatement<'conn> {
     }
 
     fn open_cursor(self, _client: &SpiClient<'conn>, args: Self::Arguments) -> SpiCursor<'conn> {
-        let args = args.unwrap_or_default();
-
-        let (mut datums, nulls): (Vec<_>, Vec<_>) = args.into_iter().map(prepare_datum).unzip();
+        let (mut datums, nulls) = self.args_to_datums(args).unwrap();
```

Comment:
specifically the FIXME right here.

Also alternatively: add `fn try_open_cursor`, then tag *this* one with `#[deprecated(since = "0.12.2", note = "lol ub")]`, and then replace the inner impl of this with just `self.try_open_cursor(_client, args).unwrap()`.

Reply:
Will write the new method too then, that's not an issue for me. And actually, I have other plans for improvements there and around (: